src/foremast/awslambda/awslambda.py

```diff
@@ -59,6 +59,7 @@ def __init__(self, app, env, region, prop_path):
         self.memory = app['lambda_memory']
         self.role = app.get('lambda_role') or generated.iam()['lambda_role']
         self.timeout = app['lambda_timeout']
+        self.concurrency_limit = app.get('lambda_concurrency_limit')
```

Comment:
Can you add a default value to `templates/configs/configs.json.j2` for this new value? It is not totally necessary, but I try to document all options in the templates.

Reply:
Sure - done.